mujoco_warp/_src/collision_primitive.py

```diff
@@ -144,14 +141,53 @@ def _geom(
   return geom
 
 
+@wp.func
+def _write_contact_ext(
```

Comment:
this function and the one below look similar?

we should include docstrings for this function and the one below to better distinguish between the utility of each function